tests/integration/devfile/docker/cmd_docker_devfile_url_test.go

```diff
@@ -194,7 +156,7 @@ var _ = Describe("odo docker devfile url command tests", func() {
 	})
 
 	Context("Describing urls", func() {
-		It("should describe URL with appropriate appropriate", func() {
+		It("should describe URL with appropriate state", func() {
 			var stdout string
```

Comment:
Instead of declaring ```stdout``` make it a single line assignment directly.

Reply:
no changes on this line